### DISCUSSION TITLE
Issue with Installation - Logs attached

### DISCUSSION THREAD

**Discussion by BzaSenshi** [Security-Onion-Solutions/securityonion#12662] [2.4]
### Version

2.4.60

### Installation Method

Security Onion ISO image

### Description

installation

### Installation Type

Eval

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

16

### RAM

32gb

### Storage for /

410gb

### Storage for /nsm

200gb

### Network Traffic Collection

other (please provide detail below)

### Network Traffic Speeds

Less than 1Gbps

### Status

No, one or more services are failed (please provide detail below)

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

Hey! I have been attempting to install Security Onion multiple times, everytime doing it (tried eval, standalone, and others but for this discussion I'm specifically talking about my latest attempt, eval) I have came up with installation errors, despite per the guide/manuals, it's all correct. From this recent failure, I have grabbed the log, and it gives a plethora of errors, and as someone who isn't mega proficient in this area, it has stunned me and I don't know what to do

Any help would be extremely appreciated, i mean that whole-ly. I have been trying to days now to try and get anything to work and I can't get it to; I'm at my wits end.

P.S. I read the guidelines for posting, but theres none that seem to be relevant to me apart from this one but I'm having to answer the questions "Network Traffic Collection" and below on this one. I'm sorry for this.

[Security Onion error.txt](https://github.com/Security-Onion-Solutions/securityonion/files/14756727/Security.Onion.error.txt)

if you do not want to click that link:

`[ERROR] Cannot create cache module directory /var/cache/salt/minon/extmods/grains. Check permissions
	result:false

Comment: One or more requisite failed: ssl.inf luxdb_crt
Result: False
Result: False
Comment: One or more requisite failed: ssl.redis_crt
Result: False
Result: False
Comment: One or more requisite failed: ssl.etc_elasticfleet_crt
Result: False
Result: False
Comment: One or more requisite failed: ssl.etc elasticfleet_logstash_crt
Result: False
Result: False
Comment: One or more requisite failed: ssl.etc_elasticfleet_logstash_key
Result: False
Comment: One or more requisite failed: ssl.etc_elasticfleet lumber jack_crt
Result: False
Result: False
Comment: One or more requisite failed: ssl.etc_elasticfleet lumber jack_key
Result: False
Comment: One or more requisite failed: ssl.etc_elasticfleet_agent_crt
Result: False
Result: False
Comment: One or more requisite failed: ssl.etc_elasticfleet_agent_key
Result: False
Comment: One or more requisite failed: ssl.etc_filebeat_crt
Result: False
Result: False
Comment: One or more requisite failed: ssl.etc_filebeat_key
Result: False
Comment: One or more requisite failed: ssl.registry_crt
Result: False
Result: False
Comment: One or more requisite failed: ssl./etc/pki/elasticsearch.crt`

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by cm-ops** [comment]
Do you have your `/root/sosetup.log`? Do you see any issues in there?

**Reply by NeoLife-iT** [reply]
Please take a look @ my discussion it might help you if you run Hyper-V with VLAN & DELL Servers

https://github.com/Security-Onion-Solutions/securityonion/discussions/12525
